packages/server/src/payloads.ts

```diff
@@ -124,7 +124,7 @@ const getArticleEpicTests = async (mvtId: number, isForcingTest: boolean): Promi
             return [...holdback];
         }
 
-        return [...regular, fallbackEpicTest];
+        return [...epicChoiceCardsTests, ...regular, fallbackEpicTest];
```

Comment:
can you also add `epicChoiceCardsTests` to the array on line 119 please? This will enable us to force show specific variants

Reply:
Done ✅